validator/accounts/accounts_activation.go

```diff
@@ -0,0 +1,289 @@
+package accounts
```

Comment:
Let's rename this file to accounts_enable_disable.go to align with the other files in the package